pylintrc

```diff
@@ -0,0 +1,585 @@
+[MASTER]
+
+# A comma-separated list of package or module names from where C extensions may
+# be loaded. Extensions are loading into the active Python interpreter and may
+# run arbitrary code.
+extension-pkg-whitelist=
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS
+
+
+# Add files or directories matching the regex patterns to the blacklist. The
+# regex matches against base names, not paths.
+ignore-patterns=
```

Comment:
As highlighted by @nymous earlier, this file is way too detailed. Here is my proposition for the pylintrc, let me know if you're ok with it:

```
[MASTER]
extension-pkg-whitelist=pydantic

[MESSAGES CONTROL]
disable=bad-continuation,
        line-too-long,
        too-few-public-methods
```